### PR TITLE
Allow command or shortcut entries more than five

### PR DESCRIPTION
**Pull request by mpyw**
This pull request updates the `terraform-provider-slackapp` to allow more than five command or shortcut entries. The key changes include:

1. Warning Diagnostics Extension
    - Added an extension for warning diagnostics:<br>`validatordiagx/diag.go`
2. `SizeAtMostWarning` Validator
    - Implemented a new validator to provide warnings for large numbers of entries:<br>`listvalidatorx/size_at_most_warning.go`
3. Relaxed Size Restrictions: 
    - Removed the strict limit of five for shortcut and command entries, offering greater flexibility.

### CODE REVIEW

File: internal/provider/datasources/slackappmanifest/features.go [hunk 1]
```diff
@@ -129,7 +130,7 @@ func (*Shortcut) schema() *schema.ListNestedBlock {
 			},
 		},
 		Validators: []validator.List{
-			listvalidator.SizeAtMost(5),
+			listvalidatorx.SizeAtMostWarning(5),
 		},
```
Comment (mpyw) [inline internal/provider/datasources/slackappmanifest/features.go:134]: Possibly need some comments

File: internal/provider/datasources/slackappmanifest/features.go [hunk 2]
```diff
@@ -189,7 +190,7 @@ func (*SlashCommand) schema() *schema.ListNestedBlock {
 			},
 		},
 		Validators: []validator.List{
-			listvalidator.SizeAtMost(5),
```
Comment (mpyw) [inline internal/provider/datasources/slackappmanifest/features.go:192]: Possibly need some comments

File: internal/provider/datasources/slackappmanifest/features.go [hunk 3]
```diff
@@ -189,7 +190,7 @@ func (*SlashCommand) schema() *schema.ListNestedBlock {
...
+			listvalidatorx.SizeAtMostWarning(5),
 		},
```
Comment (mpyw) [inline internal/provider/datasources/slackappmanifest/features.go:194]: Possibly need some comments

**Comment by siketyan** [comment]
Thank you ❤️ 

image

**Pull request merged**